FourthHomeWork/ISorting.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace FourthHomeWork
+{
+    public interface ISorting
+    {
+        void Sort()
```

Comment:
Should look like: void Sort()
Without empty body and as a result your methods void ISorting.Sort() will look like Sort()